.bashrc

```diff
@@ -80,8 +82,38 @@ if [ -x /usr/bin/dircolors ]; then
     alias egrep='egrep --color=auto'
 fi
 
+# Initiate virtualenv
 WORKON_HOME="/home/vagrant/.virtualenvs"
 source /usr/local/bin/virtualenvwrapper.sh > /dev/null 2>&1
 cd /vagrant
 
-# the workon command to be added during provisioning 
+# Ask for git configuration data from user based on input
+chmod 755 /home/vagrant/gitconfig
```

Comment:
So you want to just add ~/.gitconfig to the path? I think we need to choose another name because there is a file that is called ~/.gitconfig. I may call it ~/.configgit or something.